SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetBitReader.cpp

```diff
@@ -40,7 +40,6 @@ void FSpatialNetBitReader::DeserializeObjectRef(FUnrealObjectRef& ObjectRef)
 	}
 
 	SerializeBits(&ObjectRef.bNoLoadOnClient, 1);
-	SerializeBits(&ObjectRef.bUseSingletonClassPath, 1);
```

Comment:
deleted this component property

Reply:
didn't this get added back in as `use_class_path_to_load_object`

Reply:
ayay